app/views/editor/wikiRich.html.erb

```diff
@@ -128,6 +147,10 @@
 
 
 <script>
+  $(window).on('load',function(){
+    $('#wikiModal').modal('show');
```

Comment:
Is it possible that the modal without the `fade` class, or aria-hidden=true, might display already by default?

Reply:
@jywarren that's needed to display the modal every time the page is loaded. 
I tried removing the `fade` class and `aria-hidden=true` but it doesn't display by default :cry: